reference/carts.sf.yml

```diff
@@ -19,8 +22,10 @@ paths:
         Returns a *Cart*.
 
         <!-- theme: info -->
+
         > #### Note
-        > The Send a Test Request feature is not currently supported for this endpoint.
+        > * The `yourstore.example.com` is the store domain specific to your storefront. 
```

Comment:
```suggestion
        > * The URL `yourstore.example.com` is the store domain specific to your storefront. 
```

Reply:
@bc-andreadao Ok, sounds good. I am working on this now. I will also update Shipping Provider API.